app/templates/Event/details.html.twig

```diff
@@ -43,6 +43,19 @@
                     Event Website: <a href="{{ event.getWebsiteAddress }}" rel="nofollow">{{ event.getWebsiteAddress }}</a> <i class="fa fa-external-link-square"></i>
                     <br>
                     Short URL: <a href="{{ shortUrlForEvent(event.getStub) }}">{{ shortUrlForEvent(event.getStub) }}</a> <i class="fa fa-external-link-square"></i>
+                    {% if event.getCfpStatus %}
+                        <br>
+                        Call for Papers Status: {{ event.getCfpStatus }}
+                    {% endif %}
+                    {% if event.getCfpStatus == 'Open' %}
+                        <br>
+                        (ends {{ dateFormat(event.getCfPEndDate, 'd.m.Y') }})
```

Comment:
Twig has a [date filter](http://twig.sensiolabs.org/doc/filters/date.html) which would be more appropriate to use if it works with DateTime objects.